src/orchard/java.clj

```diff
@@ -345,28 +345,48 @@
           (first ms)
           {:candidates (zipmap (map :class ms) ms)})))))
 
+(def backported-javadoc-bases
+  "Copied from clojure.java.javadoc. These are the base urls for
+  javadocs from `clojure.java.javadoc/*core-java-api*`. It is here for
+  two reasons:
+  1. add java 13 to this list
+  2. backport for older Clojures"
+  {8 "https://docs.oracle.com/javase/8/docs/api/"
+   9 "https://docs.oracle.com/javase/9/docs/api/"
+   10 "https://docs.oracle.com/javase/10/docs/api/"
+   11 "https://docs.oracle.com/en/java/javase/11/docs/api/"
+   12 "https://docs.oracle.com/en/java/javase/12/docs/api/"
+   13 "https://docs.oracle.com/en/java/javase/13/docs/api/"})
```

Comment:
Probably we should also add some fallback for unreleased versions - I assume they are going to have the same URL format as JDK 11+.

Reply:
I looked at 14 is quite different: 

14:
https://download.java.net/java/GA/jdk14/docs/api/java.base/java/lang/String.html 

13:
https://docs.oracle.com/en/java/javase/13/docs/api/java.base/java/lang/String.html

I'm not sure if 14 is still in some kind of preview or if this is a new schema